plugins/inputs/mongodb/mongostat.go

```diff
@@ -659,6 +661,8 @@ func NewStatLine(oldMongo, newMongo MongoStatus, key string, all bool, sampleSec
 		Faults:    -1,
 	}
 
+	returnVal.UptimeMillis = newStat.UptimeMillis
```

Comment:
Looks good, but can you update the field to be in nanoseconds and rename the field to `uptime_ns` to match up with `flushes_total_time_ns`?  See line 742

Reply:
I get it why it's better to name things consistently, but I have doubts:
1. It may be misleading to have `_ns` suffix on something that has millisecond resolution.
2. It's clearer for user when metric name matches with MongoDB's command output.

Reply:
These are good points but I'd still like to make this change for consistency.  We are trying to do this across the board for all duration and time values and nanosecond precision gives us the most flexibility and is consistend with our timestamp format.

Reply:
Replaced `uptime_ms` with `uptime_ns`.